scripts/build_ascent/build_ascent.sh

```diff
@@ -653,7 +653,7 @@ cmake -S ${kokkos_src_dir} -B ${kokkos_build_dir} ${cmake_compiler_settings} \
 
 echo "**** Building Kokkos ${kokkos_version}"
 cmake --build ${kokkos_build_dir} --config ${build_config} -j${build_jobs}
-echo "**** Installing VTK-m ${kokkos_version}"
+echo "**** Installing Viskores ${kokkos_version}"
```

Comment:
```suggestion
echo "**** Installing Kokkos ${kokkos_version}"
```
This looks like a result of a typo by me earlier. This is Kokkos not VTK-m / Viskores